src/vs/editor/browser/view.ts

```diff
@@ -111,6 +111,8 @@ export class View extends ViewEventHandler {
 	private _shouldRecomputeGlyphMarginLanes: boolean = false;
 	private _renderAnimationFrame: IDisposable | null;
 
+	private _targetWindow: CodeWindow | undefined;
```

Comment:
@aiday-mar lets do a follow up to only store the `vscodeWindowId` (of type `number`) and not the actual `CodeWindow`. I am worried about potential memory leaks here where we hold onto the window longer than needed.

Reply:
Maybe even that info could be stored within the `NativeEditContext` since its only really needed there.

Reply:
Yep sounds good, I'll make a PR to change that.